projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -258,112 +274,120 @@ const BarChartInternal: FC< BarChartProps > = ( {
-							yAccessor={ chartOptions.accessors.yAccessor }
-							xAccessor={ chartOptions.accessors.xAccessor }
-							colorAccessor={ getBarBackground( index ) }
+					{ withPatterns && (
+						<>
+							<defs data-testid="bar-chart-patterns">
+								{ dataSorted.map( ( seriesData, index ) =>
+									renderPattern( index, getColor( seriesData, index ) )
+								) }
+							</defs>
+							<style>
+								{ dataSorted.map( ( seriesData, index ) =>
+									createPatternBorderStyle( index, getColor( seriesData, index ) )
+								) }
+							</style>
+						</>
+					) }
+
+					{ highlightedBarStyle && <style>{ highlightedBarStyle }</style> }
+
+					<BarGroup padding={ chartOptions.barGroup.padding }>
+						{ dataWithVisibleZeros.map( ( seriesData: SeriesData, index: number ) => (
+							<BarSeries
+								key={ seriesData?.label }
+								dataKey={ seriesData?.label }
+								data={ seriesData.data as DataPointDate[] }
+								yAccessor={ chartOptions.accessors.yAccessor }
+								xAccessor={ chartOptions.accessors.xAccessor }
+								colorAccessor={ getBarBackground( index ) }
+							/>
+						) ) }
+					</BarGroup>
+
+					<Axis { ...chartOptions.axis.x } />
+					<Axis { ...chartOptions.axis.y } />
+
+					{ withTooltips && (
+						<AccessibleTooltip
+							detectBounds
+							snapTooltipToDatumX
+							snapTooltipToDatumY
+							renderTooltip={ renderTooltip || renderDefaultTooltip }
+							selectedIndex={ selectedIndex }
+							tooltipRef={ tooltipRef }
+							keyboardFocusedClassName={ styles[ 'bar-chart__tooltip--keyboard-focused' ] }
+							series={ data }
+							mode="individual"
 						/>
-					) ) }
-				</BarGroup>
-
-				<Axis { ...chartOptions.axis.x } />
-				<Axis { ...chartOptions.axis.y } />
-
-				{ withTooltips && (
-					<AccessibleTooltip
-						detectBounds
-						snapTooltipToDatumX
-						snapTooltipToDatumY
-						renderTooltip={ renderTooltip || renderDefaultTooltip }
-						selectedIndex={ selectedIndex }
-						tooltipRef={ tooltipRef }
-						keyboardFocusedClassName={ styles[ 'bar-chart__tooltip--keyboard-focused' ] }
-						series={ data }
-						mode="individual"
+					) }
+				</XYChart>
+
+				{ showLegend && (
+					<Legend
+						items={ legendItems }
+						orientation={ legendOrientation }
+						alignmentHorizontal={ legendAlignmentHorizontal }
+						alignmentVertical={ legendAlignmentVertical }
+						className={ styles[ 'bar-chart__legend' ] }
+						shape={ legendShape }
+						ref={ legendRef }
+						chartId={ chartId }
 					/>
 				) }
-			</XYChart>
-
-			{ showLegend && (
-				<Legend
-					items={ legendItems }
-					orientation={ legendOrientation }
-					alignmentHorizontal={ legendAlignmentHorizontal }
-					alignmentVertical={ legendAlignmentVertical }
-					className={ styles[ 'bar-chart__legend' ] }
-					shape={ legendShape }
-					ref={ legendRef }
-					chartId={ chartId }
-				/>
-			) }
-		</div>
+			</div>
```

Comment:
Where are the `{ children }` 😄

Reply:
Very good question 😆  they got left behind! Adding back in now.